src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+export const categoriesSlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state.status,
+  },
+});
```

Comment:
- [ ] Great work so far! However, according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md), the reducer inside the categories slice needs to check the status and update the state. It appears that this step has been missed in your code, so please ensure that you check the payload from the action before making any updates to the state, Here is an example of how to do this:
```jsx
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories

```
- [ ] As you might've guessed this also means your initial state only needs one property called `categories` that holds an empty array as its initial value. So please consider removing the `status` property. 😉